src/network/utils.ts

```diff
@@ -59,13 +47,6 @@ function isPort(port: any, connect: boolean = false): port is Port {
   return true;
 }
 
-/**
- * Given a bearer token, return a authentication token string.
- */
-function toAuthToken(token: string) {
-  return `Basic ${Buffer.from(token).toString('base64')}`;
-}
-
```

Comment:
This authentication is done by the client middleware. The utils for that were moved to the client domain.

Reply:
Is it still the same thing though? That is using an authentication token like `Basic ...`? That was to replicate "bearer" tokens in the HTTP standard. Are we doing that now in the `metadata` property in our RPC messages? I want to preserve the `Basic` and `base64` option. Furthermore is `base64` correct or should it have been changed to `base64url`?

Reply:
Yes, it follows the same format and structure. It's under `metadata.authorization`. Have a look at `src/client/utils/utils.ts`, it was moved there.

Reply:
I see, when moving things around, this stuff should be documented in the MRs too, otherwise my mental map of where things are gets lost. Diagrams are the best way of communicating this.